docs/getting-started/playground-start.md

```diff
@@ -0,0 +1,105 @@
+---
+id: playground-start
+---
+
+# Gno Playground
+
+## Overview
+
+The Gno Playground is an innovative web-based editor and sandbox that enables developers to 
+interactively work with the Gno language. It makes coding, testing,
+and deploying simple with its diverse set of tools and features. Users can
+share code, run tests, and deploy projects to Gno.land networks, 
+making it the perfect tool to get started with Gno development.
+
+## Prerequisites
+
+- **Internet connection**
+- **A Gno.land compatible wallet**
+  - Currently, [Adena](https://www.adena.app/) is the go-to wallet for
+Gno.land, with more wallets being introduced in the future.
+
+## Playground Features
+
+To get started, visit the Playground at [play.gno.land](https://play.gno.land). You will be greeted with a
+simple `package.gno` file:
+
+![default_playground](../assets/getting-started/playground/default_playground.png)
+
+The Playground has the following features:
+- `Share` - Generate a unique, short, and shareable identifier for your Gno code.
+- `Deploy` - Connect your wallet and publish your code on Gno.land.
+- `Format` - Automatically adjust your Gno code's structure and style for optimal readability and consistency.
+- `Run` - Execute a particular expression within your code to validate its functionality and output.
```

Comment:
What do you think about expanding a bit more on this functionality? Specifically on what `expression` means

Reply:
I would like to keep this specific item list relatively short, and expand on it in the individual feature sections.